.github/workflows/release.yaml

```diff
@@ -0,0 +1,25 @@
+name: Release charm to other tracks and channels
```

Comment:
In the observability team, we've opted for a somewhat different approach that I think is less likely to blow up due to erroneous user input. It also has the benefit of being more aligned with the idea of not skipping stages: 

https://github.com/canonical/grafana-agent-k8s-operator/blob/709f6a743db4c314e0760316c27578296b819d85/.github/workflows/promote.yaml#L14-L38